src/update/NoUpdatesAvailableDialog.cpp

```diff
@@ -0,0 +1,96 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file NoUpdatesAvailableDialog.cpp
+  @brief Define a dialog with Information about no updates available.
+
+  Anton Gerasimov
+
+**********************************************************************/
+
+#include "NoUpdatesAvailableDialog.h"
+
+#include "widgets/HelpSystem.h"
+#include "ShuttleGui.h"
+
+#include "prefs/PrefsDialog.h"
+#include "ui/AccessibleLinksFormatter.h"
+
+#include <wx/stattext.h>
+
+static const auto title =
+/* i18n-hint: Title of the dialog no updates available. */
+XO("No updates available");
+
+BEGIN_EVENT_TABLE(NoUpdatesAvailableDialog, wxDialogWrapper)
+    EVT_BUTTON(wxID_OK, NoUpdatesAvailableDialog::OnOk)
+END_EVENT_TABLE()
+
+NoUpdatesAvailableDialog::NoUpdatesAvailableDialog(wxWindow* parent)
+    : wxDialogWrapper(
+         parent, -1, XO("Check for Updates"), wxDefaultPosition, wxDefaultSize, wxCAPTION | wxCLOSE_BOX)
+{
+   ShuttleGui S(this, eIsCreating);
+
+   S.SetBorder(8);
+
+   S.StartVerticalLay();
+   {
+      S.AddSpace(0, 12);
+
+      S.StartHorizontalLay();
+      {
+         S.AddSpace(12, 0);
+
+         S.StartVerticalLay();
+         {
+            wxStaticText* titleCtrl = S.AddVariableText(title, false, 0, 500);
+
+            wxFont font = titleCtrl->GetFont().MakeLarger().MakeBold();
+
+            titleCtrl->SetFont(font);
+
+            AccessibleLinksFormatter preferencesMessage(
+               /* i18n-hint: %s is replaced with 'Preferences > Application'. */
+               XO("You are on the latest version of Audacity. Consider turning %s on in Preferences, if you want to be informed automatically about new versions."));
```

Comment:
Maybe the message should only contain the first sentence, in the case that this dialog opens in the case of automatic update checking that the user has already enabled.

So you could check the state of`DefaultUpdatesCheckingFlag` here.

Reply:
I think we should keep of design, because in case when user forgot current update preferences state, it easy and fast way to move to update preferences and check it.

Reply:
No, consider.  If we can tell the preference is enabled, it is silly to suggest that the user enables it.  If this code executes because if automatic checking, not the help menu item, then the preference must have been on.

Reply:
But what you think about case when user forget current update settings state? Just sometimes, I use the same fast link in other applications, that I long time don't using and want to do (just not rememder where update settings was described), I use fast way to check new version and check update settings.

Or course this is may only my case, but may be someone using this way like as me, for fast redirecting to settings. What you think about this?

Reply:
Are you saying the reminder should suggest turning the preference on or off?  Instead of what the words say now.

Reply:
Yes

Reply:
Either:

A) Let's do what Paul suggests, ie. check if the user has automatic updates on, and show a version of this screen without the text reference to turning automatic updates on
OR

B) we could change that text to be more generic, and show the same text for everyone, whether or not they have automatic updates enabled. ie. "If you want to turn your preference for automatic updates on or off, you can find it in Preferences (link)"